src/Redux/Actions/readArticleActions.js

```diff
@@ -0,0 +1,33 @@
+import types from './index';
+import axios from '../../configs/axios';
+
+const { READ_ARTICLE, GET_ARTICLE_TAGS, READ_ARTICLE_ERROR } = types;
+
+export default {
+  readArticle: slug => (dispatch) => {
```

Comment:
This is a reusable action and please change it to make so...because your just fetch article related content so make it a get article action @MCFrank16